src/wc-demo.html

```diff
@@ -0,0 +1,24 @@
+<meta charset="utf-8">
```

Comment:
Since this is a great minimal example, what do you think about moving this file to the README, in a future documentation section (I take care of that) and use the site (come-partecipo.italia.it) itself as a working real world example?